CalculatorApp/Calculator.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorApp
+{
+    public class Calculator
+    {
+        public int add(string numbers)
+        {
+            int sum = 0;
+            if (string.IsNullOrEmpty(numbers) || numbers.Length == 0)
+            {
+                return sum;
+            }
+            bool IsValidInput = ValidateInput(numbers);
```

Comment:
var name should be in camelCase 
and use var instead of bool 

apply to all vars in the PR